src/Windowing/OpenToolkit.Windowing.Common/Enums/OpenGLVersion.cs

```diff
@@ -0,0 +1,27 @@
+//
+// OpenGLVersion.cs
+//
+// Copyright (C) 2019 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+namespace OpenToolkit.Windowing.Common
+{
+    /// <summary>
+    /// Describes the OpenGL version you want the window to use.
+    /// </summary>
+    public enum OpenGLVersion
+    {
+        /// <summary>
+        /// Selects core profile OpenGL 3.3
+        /// </summary>
+        OPENGL_3_3_CORE,
```

Comment:
I think it would be nice to use version information instead of a enum containing version info. Additionaly it would be nice to directly select Vulkan as well ;)
Next thing I think which would be really nice, would be a version range, so that you can say what minimal version you need, and which maximum version, and it tries to create a context in that version range and if nothing is available it throws an exception...

Reply:
Closing this review because the discussion is happening on Discord instead